package.json

```diff
@@ -13,7 +13,8 @@
     "test": "ng test",
     "test:headless": "ng test --watch=false --browsers=ChromeHeadless",
     "lint": "ng lint",
-    "format": "prettier --write \"**/*.{js,ts,html}\""
+    "format": "prettier --check \"**/*.{js,ts,html,css,scss}\"",
```

Comment:
This will also format generated proto files, I think? Maybe restrict to src/ ?

Reply:
> This will also format generated proto files, I think? Maybe restrict to src/ ?

No, `prettier` is actually surprisingly intelligent (or obtuse, depending on how you look at it). It considers `.gitignore` and `.prettierignore` files. You exclude all the generated stuff in `.gitignore`, so `prettier` wouldn't touch it. However, if files starting with the dot get lost, you run into trouble... like I did.